.github/workflows/ci.yml

```diff
@@ -101,69 +101,40 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v4
-      - name: Install PHP
-        uses: shivammathur/setup-php@v2
-        with:
-          php-version: "${{ matrix.php-version }}"
-          ini-file: development
-          coverage: none
-          tools: composer:v2
-      - name: Show Composer version
```

Comment:
Please let's keep these two steps (i.e., switch them to `runTests.sh`) to help debugging Composer problems.

Reply:
I've just tested it. It looks like we cannot check the version, i.e., this does _not_ work:

```bash
./Build/Scripts/runTests.sh -s composer --version
```

Outputting the configuration works fine, though:

```bash
./Build/Scripts/runTests.sh -s composer config --global --list
```

So let's drop outputting the Composer version and keep/adapt outputting the Composer configuration.